assets/components/svgs/mastercardLogo.jsx

```diff
@@ -11,7 +11,9 @@ export default function SvgMastercardLogo() {
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 36 24"
       preserveAspectRatio="xMinYMid"
+      aria-labelledby="svg-mastercard-logo-title"
     >
+      <title id="svg-mastercard-logo-title">Mastercard</title>
```

Comment:
ditto the Facebook comment

Reply:
The surrounding block has an a11y hint explaining what types of payments we support. This is just extra information if a user wants to drill down into the image.